paddle/fluid/operators/rank_attention_op.cu

```diff
@@ -243,10 +243,16 @@ class RankAttentionGradOpCUDAKernel : public framework::OpKernel<T> {
 
 namespace ops = paddle::operators;
 using GPUCtx = phi::GPUContext;
-REGISTER_OP_CUDA_KERNEL(rank_attention,
```

Comment:
Maybe `line 245` could also be deleted
`using GPUCtx = phi::GPUContext;`

Reply:
3q, I will do it in next pr.